src/bin/lfortran.cpp

```diff
@@ -2268,7 +2269,7 @@ int main_app(int argc, char *argv[]) {
     app.add_flag("--ignore-pragma", compiler_options.ignore_pragma, "Ignores all the pragmas");
     app.add_flag("--stack-arrays", compiler_options.stack_arrays, "Allocate memory for arrays on stack");
     app.add_flag("--wasm-html", compiler_options.wasm_html, "Generate HTML file using emscripten for LLVM->WASM");
-    app.add_flag("--experimental-simplifier", compiler_options.po.experimental_simplifier, "Use experimental simplifier pass");
+    app.add_flag("--no-experimental-simplifier", no_experimental_simplifier, "Do not use experimental simplifier pass");
```

Comment:
If we merge this PR, it's not an experimental simplifier anymore, right?
```suggestion
    app.add_flag("--no-default-simplifier", no_experimental_simplifier, "Do not use the default simplifier pass");
```